frontend/cypress/integration/featureFiles/overview.feature

```diff
@@ -120,5 +120,5 @@ Feature: Kiali Overview page
     Then user sees the "istio-system" namespace card
     And user sees the "Control plane" label in the "istio-system" namespace card
     And user sees the "Outbound policy" label in the "istio-system" namespace card
-    When user clicks the toggle on the right side of the "istio-system" namespace card
-    Then user can see links to external services
```

Comment:
This fix is now just testing the kebab exists instead of clicking on it and ensuring the link exists? Fine by me but tagging @matejnesuta since I think he initially added the test. Is the `Then user can see links to external services` definition used anywhere else or should that be removed as well?

Reply:
It simplifies the test to avoid the flaky. I talked with @jshaughn and this approach was done in other test I guess. Having said that, sure, I also waits for @matejnesuta's opinion. No problem.

Reply:
Hi, @matejnesuta do you have any comments about this?.

Reply:
Hi @leandroberetta. Sorry for the late reply, I was out due to my final exams.

Anyways. As Nick mentioned, the original idea was to test that the menu has some additional links, compared to other menus. That seemed to work originally, but looking at it now, it might've not been the best idea. I am honestly not sure, if it's necessary to check the existence of the kebab only. It seems kind of pointless to me, but I suppose you might see some benefits in doing that and it is only one assertion anyway.

Reply:
In the past we've had trouble getting kebab tests to work. My guess is that after you click the kebab there's a refresh that happens _before_ the links are found on the screen which causes the kebab menu to close and the links to not be found.

Anyways some tests that are reliable are better than more tests that are unreliable so I'd prefer we remove the kebab link assertions for now until we can figure out how to test those reliably.